bin/check-es-indices-sizes.rb

```diff
@@ -159,12 +159,11 @@ def build_indices_with_sizes
   def run
     node_fs_stats = client.nodes.stats metric: 'fs,indices'
     nodes_being_used = node_fs_stats['nodes'].values.select { |node| node['indices']['store']['size_in_bytes'] > 0 }
-    # rubocop:disable SingleLineBlockParams
-    # rubocop:disable LineLength
-    used_in_bytes = nodes_being_used.map { |node| node['fs']['data'].map { |data| data['total_in_bytes'] - data['available_in_bytes'] }.flatten }.flatten.inject { |sum, x| sum + x }
-    # rubocop:enable LineLength
+    # TODO: come back and cleanup all these rubocop disables with a little refactor
```

Comment:
I know this is not the code that was changed but somehow rubocop started complaining. This is super ugly and hard to read, we should come back and refactor this later.